Digital Shadows/digital_shadows_searchlight/_meta/manifest.yml

```diff
@@ -0,0 +1,5 @@
+uuid: ccf942fe-c839-42be-a081-5c3f946e80f5
+name: digital_shadows_searchlight
+slug: digital_shadows_searchlight
+description: >-
+   Your security team is busy. Digital Shadows SearchLight removes the noise of Threat Intelligence, making it easier to identify what is important so you can make better decisions faster.
```

Comment:
This description must includes the sentence that details how SEKOIA.IO can leverage this integration to improve the security of its customers.